src/ApparentHorizons/SpherePackIterator.hpp

```diff
@@ -0,0 +1,106 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///
+/// \file
+/// Defines class SpherePackIterator.
+
+//
+// iterate through an array with the layout of the Ylm-spectral
+// coefficients of SpherePack.
+//================================================================
+#pragma once
+
+#include <ostream>
+#include <vector>
+
+/// Iterates through the coefficients of a SpherePack spectral
+/// transform.  The internal SpherePack ordering is not intuitive, so
+/// SpherePackIterator exists for the purpose of iterating over the
+/// array and determining which specific values of l and m correspond
+/// to each iteration.
+///
+/// \detail
+/// SpherePackIterator iterates over each coefficient ONCE, in some
+/// arbitrary order. There are member functions to determine the values
```

Comment:
Remove (or move if they are not below) this last part about the member functions